src/lib/storage/yaml_writer.rb

```diff
@@ -139,7 +158,7 @@ def yaml_partition(partition)
           "size" => DiskSize.new(partition.region.to_kb(partition.region.length)).to_s,
           "name" => partition.name,
           "type" => @partition_types[partition.type],
-          "id"   => @partition_ids[partition.id]
+          "id"   => @partition_ids[partition.id] || "0x#{partition.id.to_s(16)}"
```

Comment:
ok, make sense, in such case I really think it is better to have different map name like `@partition_symbolic_names`